src/sbt-test/sbt-1.5/scala-3/project/build.properties

```diff
@@ -0,0 +1 @@
+sbt.version=1.5.2
```

Comment:
The way scripted are tested against different versions of sbt is quite messy at the moment, but I would rather wait for https://github.com/scalacenter/scalafix/issues/1333 to clean it up.